.circleci/scripts/binary_checkout.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
For the tracking purposes, please specify where this file is from https://github.com/pytorch/pytorch/blob/2c7df1360aa17d4a6d6726998eede3671bcb36ee/.circleci/scripts/binary_populate_env.sh
```suggestion
#!/bin/bash
# Derived from https://github.com/pytorch/pytorch/blob/2c7df1360aa17d4a6d6726998eede3671bcb36ee/.circleci/scripts/binary_populate_env.sh
```